src/Compilers/CSharp/Portable/Symbols/Retargeting/RetargetingParameterSymbol.cs

```diff
@@ -273,6 +275,8 @@ public RetargetingPropertyParameterSymbol(RetargetingPropertySymbol retargetingP
             _retargetingProperty = retargetingProperty;
         }
 
+        internal override bool IsReadOnly => _retargetingProperty.IsReadOnly;
```

Comment:
This doesn't feel right. Is there a relationship between readonly-ness of a property and readonly-ness of its parameters? Looks like a unit-test is in order.
Consider moving implementation to the base class of RetargetingMethodParameterSymbol and RetargetingPropertyParameterSymbol.